agent/04_agent_configure.sh

```diff
@@ -522,6 +504,10 @@ if [ ! -z "${MIRROR_IMAGES}" ]; then
 
 fi
 
+if [ "${OC_MIRROR}" == "true " ] && [  "${MIRROR_MCE}" == "true " ]; then
```

Comment:
Right now disconnected scenario is identified by the config variable `MIRROR_IMAGES`, so the decision of mirroring (or not) the dependencies required by MCE should be made primarily using such var.
After that, the decision on which approach to be used for performing the mirror could be done by evaluating `OC_MIRROR`. Eventually, if we don't want to support the legacy way of mirroring the build then should fail in that case.

Reply:
I believe that mce requires the OC_MIRROR implementation as it allows an ImageSet to be build with the proper operators.

Reply:
I'll make a follow up PR to try to refactor to the following:

```
export MIRROR_IMAGES=true
export MIRROR_OLM=multicluster-engine,local-storage-operator
export OC_MIRROR=true
export AGENT_DEPLOY_MCE=true
```